pullRequestHandler_test.go

```diff
@@ -95,3 +95,34 @@ func Test_hasNoDcoLabel(t *testing.T) {
 		})
 	}
 }
+func Test_firstTimeContributor(t *testing.T) {
+	var authorLabel = []struct {
+		label        string
+		expectedBool bool
+	}{
+		{
+			label:        "NONE",
+			expectedBool: true,
+		},
+		{
```

Comment:
Is this test-case duplicated?

Reply:
Please see this feedback @mrtind

Reply:
I saw it and i changed it.But i have problems with updating my branch with master and pushing the changes in the fork.